src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.PropertyTests.cs

```diff
@@ -152,6 +152,50 @@ private static IEnumerable<TElement> KMaxElements<TElement>(TElement[] elements,
             }
         }
 
+        [Theory]
+        [MemberData(nameof(GetRandomStringArrays))]
+        public static void KMaxElements_DequeueEnqueue_String(string[] elements)
+        {
+            const int k = 5;
+            KMaxElements_DequeueEnqueue(elements, k, s_stringComparer);
+        }
+
+        [Theory]
+        [MemberData(nameof(GetRandomIntArrays))]
+        public static void KMaxElements_DequeueEnqueue_Int(int[] elements)
+        {
+            const int k = 5;
+            KMaxElements_DequeueEnqueue(elements, k);
+        }
+
+        private static void KMaxElements_DequeueEnqueue<TElement>(TElement[] elements, int k, IComparer<TElement>? comparer = null)
```

Comment:
The method name is a bit of a misnomer, since it doesn't actually return `k` min elements. Since "k max elements" is just an algorithm specifically showcasing insert-then-extract, I'd be fine if we didn't anything in `PropertyTests.cs`.

Reply:
Ok. I removed the DequeueEnqueue cases from the PropertyTests.